src/ChangePassword.tsx

```diff
@@ -63,25 +65,25 @@ const ChangePassword = ({
 
   const { notify } = useContext(NotificationContext)
 
-  const checkPasswordAge = async (
-    id: number,
-    dataProvider: DataProvider
-  ): Promise<void> => {
-    const {
-      data: { lastUpdatedAt }
-    } = await dataProvider.getOne<_Users>(constants.R_USERS, {
-      id
-    })
+  // const checkPasswordAge = async (
+  //   id: number,
+  //   dataProvider: DataProvider
+  // ): Promise<void> => {
+  //   const {
+  //     data: { lastUpdatedAt }
+  //   } = await dataProvider.getOne<_Users>(constants.R_USERS, {
+  //     id
+  //   })
 
-    if (
-      lastUpdatedAt !== null &&
-      lastUpdatedAt !== '' &&
-      !isDateNotInPastDays(lastUpdatedAt, 1)
-    )
-      throw new Error(
-        'Password update not allowed. Please wait at least one day before updating your password again.'
-      )
-  }
+  //   if (
+  //     lastUpdatedAt !== null &&
+  //     lastUpdatedAt !== '' &&
+  //     !isDateNotInPastDays(lastUpdatedAt, 1)
+  //   )
+  //     throw new Error(
+  //       'Password update not allowed. Please wait at least one day before updating your password again.'
+  //     )
+  // }
```

Comment:
Do we still have the minimum password age testing?  Does it now happen somewhere else?

Reply:
We do not need this password age testing functionality. That's why I commented this. But I will remove this as well.